model-optimizer/extensions/front/tf/space_to_batch.py

```diff
@@ -79,8 +81,7 @@ def find_and_replace_pattern(self, graph: Graph):
 
             in0_rank_1d = create_op_node_with_second_input(graph, Unsqueeze, int64_array([0]),
                                                            {'name': node.name + '/1d_rank_of_0'}, in0_rank)
-            in1_rank_1d = create_op_node_with_second_input(graph, Unsqueeze, int64_array([0]),
-                                                           {'name': node.name + '/1d_rank_of_1'}, in1_rank)
+            in1_rank_1d = in1_rank
```

Comment:
The `in1_rank` variable is not needed at all. The input of this node is connected in line 87 (using the name 'in1_rank') and the output of this node is connected in line 89 (using the name `in_rank_1d`). 

Moreover, this is not a rank node. It actually gets a shape from the input port 1 of the BTS/SPB node.